packages/material-ui/package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "start": "tsdx watch",
     "build": "rimraf dist && tsdx build",
-    "test": "tsdx test"
+    "test": "tsdx test --updateSnapshot"
```

Comment:
I think you should revert this (so CI doesn't update snapshots too)

Reply:
Good catch, didn’t mean to let that sneak in. Thanks!